pkg/apis/kafka/v1alpha1/reset_offset_types_test.go

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/assert"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+)
+
+func TestResetOffset_GetGroupVersionKind(t *testing.T) {
+	resetOffset := ResetOffset{}
+	gvk := resetOffset.GetGroupVersionKind()
+	if gvk.Kind != "ResetOffset" {
+		t.Errorf("Should be 'ResetOffset'.")
+	}
```

Comment:
This also seems odd vs. using assert.Equal

Reply:
Same response as above - will either leave or refactor everything out depending on feedback - closing this discussion as duplicate...